dhctl/pkg/config/meta.go

```diff
@@ -139,6 +143,34 @@ func (m *MetaConfig) Prepare() (*MetaConfig, error) {
 	return m, nil
 }
 
+func validateRegistryDockerCfg(cfg string) error {
+	regcrd, err := base64.StdEncoding.DecodeString(cfg)
+	if err != nil {
+		return fmt.Errorf("unable to decode registryDockerCfg: %w", err)
+	}
+
+	var creds struct {
+		Auths map[string]interface{} `json:"auths"`
+	}
+
+	if err = json.Unmarshal(regcrd, &creds); err != nil {
+		return fmt.Errorf("unable to unmarshal docker credentials: %w", err)
+	}
+
+	regx, err := regexp.Compile(`^([a-z]|\d)+(\.?|\-?)(([a-z]|\d)+(\.|\-|))*([a-z]+|([a-z]|\d)\:\d+)$`)
```

Comment:
Let use net/url package for checking
https://go.dev/play/p/h9es7csl1yo

We need to che check that host is not empty port can be not empty, otherwise is empty